ts/src/base/ws/Cache.ts

```diff
@@ -221,9 +221,10 @@ class ArrayCacheBySymbolById extends ArrayCache {
 
 class ArrayCacheBySymbolBySide extends ArrayCache {
 
-    constructor () {
-        super ()
+    constructor (maxSize = undefined, hedged = true) {
```

Comment:
My concern is that most of the time we don't know if the user is currently on hedged/one-way mode 🤔 How will we handle those cases?

Reply:
good point, unfortunately we don't. In this case kucoinfutures doesn't support hedged positions yet, so we are safe. But you are right we have to be carefull for exchanges that allow both, in which case unfortunately I don't think we can know. In those cases we could allow the user to set the option in the options, however I can see how it can lead to errors.